frontend/uddhar-frontend/src/coordinator/pages/sub-page/CenterPanel.jsx

```
@@ -1,29 +1,21 @@
 import PropTypes from "prop-types";
-import DisasterSummary from "../../components/DisasterSummary";
-import OrgTable from "../../components/OrgTable";
-import WeatherCard from "../../components/WeatherCard";
+import Map from "../sub-page/Map";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)